### PR TITLE
Fix/dapp connector transaction information

### PR DESCRIPTION
**Pull request by mchappell**
# Checklist

- [x] JIRA - [LW-7024](https://input-output.atlassian.net/browse/LW-7024)
- [x] JIRA - [LW-7026](https://input-output.atlassian.net/browse/LW-7026)
- [x] JIRA - [LW-7028](https://input-output.atlassian.net/browse/LW-7028)
- [x] Screenshots added.

---

## Proposed solution

Updated styling for cip30 popup and encoded URI components for consistent display of DApp logo

## Screenshots

Screenshot 2023-06-16 at 11 05 13

Screenshot 2023-06-16 at 11 05 21

[LW-7024]: https://input-output.atlassian.net/browse/LW-7024?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LW-7026]: https://input-output.atlassian.net/browse/LW-7026?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LW-7028]: https://input-output.atlassian.net/browse/LW-7028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ✅ [test report](https://lace-qa:8PP5wBaDV6UoXj@d1terqjryk7mu9.cloudfront.net/linux/chrome/687/5409971102/index.html) for [c1e55c99](https://github.com/input-output-hk/lace/pull/136/commits/c1e55c990cd46c94d4c6fbcea42afaa7aaa816c1)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 36     | 0      | 0       | 0     | 36    | ✅     |

### CODE REVIEW

File: apps/browser-extension-wallet/src/lib/scripts/background/cip30.ts [hunk 1]
```diff
@@ -11,19 +11,20 @@ import { DAPP_CHANNELS } from '../../../utils/constants';
...
 }>(undefined);
 
-const getDappInfoFromLastActiveTab: () => Promise<Wallet.DappInfo> = async () =>
-  await getLastActiveTab().then((t) => ({
-    logo: t.favIconUrl || getRandomIcon({ id: origin, size: 40 }),
-    name: t.title || t.url.split('//')[1],
-    url: t.url.replace(/\/$/, '')
-  }));
-
+const getDappInfoFromLastActiveTab: () => Promise<Wallet.DappInfo> = async () => {
```
Comment (vetalcore) [inline apps/browser-extension-wallet/src/lib/scripts/background/cip30.ts:20]: same as in apps/browser-extension-wallet/src/lib/scripts/background/util.ts?

File: packages/core/src/ui/components/DappTransaction/DappTransaction.module.scss [hunk 2]
```diff
@@ -122,3 +118,14 @@
 :global(.__react_component_tooltip) {
   @include tooltip-default;
 }
+
+.sub {
+  font-weight: 400;
+  font-size: var(--bodySmall);
+  line-height: size_unit(3);
```
Comment (vetalcore) [inline packages/core/src/ui/components/DappTransaction/DappTransaction.module.scss:125]:
```suggestion
  @includes text-bodySmall-medium;
```

**Review by vetalcore — CHANGES_REQUESTED** [review]

File: packages/core/src/ui/components/DappTransaction/DappTransaction.module.scss (hunk 2 above)
Comment (mchappell) [reply packages/core/src/ui/components/DappTransaction/DappTransaction.module.scss]: Updated in [52e4e49](https://github.com/input-output-hk/lace/pull/136/commits/52e4e490f85379c1069eecdb6939ac86a3208a9f)

File: apps/browser-extension-wallet/src/lib/scripts/background/cip30.ts (hunk 1 above)
Comment (mchappell) [reply apps/browser-extension-wallet/src/lib/scripts/background/cip30.ts]: updated in [52e4e49](https://github.com/input-output-hk/lace/pull/136/commits/52e4e490f85379c1069eecdb6939ac86a3208a9f)

**Review by vetalcore — APPROVED** [review]

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Great work @mchappell 👍

**Pull request merged**